test/cli.test.mjs

```diff
@@ -0,0 +1,62 @@
+import chai from "chai";
+import shell from "shelljs";
+
+const expect = chai.expect;
+
+describe("Project builder", function () {
+  beforeEach(function () {
+    // create tmp dir to run the project builder in
+    shell.rm("-rf", "tmp");
+    shell.mkdir("-p", "tmp");
+    shell.cd("tmp");
+  });
+
+  afterEach(function () {
+    // cleanup
+    shell.cd("..");
+    shell.rm("-rf", "tmp");
+  });
+
+  it("should create a config file", async function () {
+    /* eslint-disable node/no-unsupported-features/es-syntax */
+
+    // The file `choose-yes.txt` has the key stokes needed that would
+    // go through the prompt and choose yes for the question being asked
+    shell.cat("../test/choose-yes.txt").exec("../dist/esm/up.js --init");
```

Comment:
piping the keystokes needed to interact with the prompt by putting the key strokes into a file and using cat.

Reply:
oh wow, this is super cool. I'm going to steal this for testing the init method on the tableland cli!